addons/courierEditor/courierInit.gd

```diff
@@ -6,7 +6,7 @@ extends Node2D
 func _ready():
 	var courierScene = load("res://addons/courierEditor/courier.tscn")
 	# TODO: Not sure about this 1, if problems,
-	var courier = courierScene.instantiate(1).duplicate(6)
+	var courier = courierScene.instantiate(1).duplicate()
```

Comment:
Is the default set of options preferable? I'm not entirely sure how `.duplicate()` works, but I thought it needed to be told to not duplicate by instantiation, so that couriers wouldn't be able to have it's own settings.

Reply:
Testing indicates no difference that I can spot, may as well leave at default with no param.